src/common/containers/build-details.js

```diff
@@ -29,23 +34,32 @@ class BuildDetails extends Component {
     if (buildFailed) {
       // if build has failed show snapcraft debug instruction
       helpBox = (
-        <HelpInstallSnap headline='To debug this build:'>
+        <HelpInstallSnapBox headline='To debug this build:'>
           sudo snap install lxd # if you don’t have LXD already<br/>
           sudo snap install --classic --edge snapcraft # if you don’t have snapcraft already<br/>
           <br/>
           git clone {repository.url}<br/>
           cd {repository.name}<br/>
           snapcraft cleanbuild --debug
-        </HelpInstallSnap>
+        </HelpInstallSnapBox>
       );
     } else if (snap && snap.store_name && build.storeRevision) {
       // otherwise if we have snap name and revision show install instructions
       helpBox = (
-        <HelpInstallSnap
-          headline='To test this build on your PC or cloud instance:'
-          name={ snap.store_name }
-          revision={ build.storeRevision }
-        />
+        <HelpBox>
+          <HelpInstallSnap
```

Comment:
Is `HelpInstallSnapBox` worth the effort if it's this easy to combine?

If it is, then maybe this should be a `HelpInstallSnapBox`.